vignettes/teal-options.Rmd

```diff
@@ -156,6 +156,12 @@ list(
 )
 ```
 
+### `teal.show_src` (`logical`)
```

Comment:
This name is deliberately different from the property to disable inside a module as it has a different goal (hides/removes it)